Sources/Sentry/Profiling/SentryProfiledTracerConcurrency.mm

```diff
@@ -55,12 +54,12 @@
 std::mutex _gStateLock;
 
 void
-trackProfilerForTracer(SentryProfiler *profiler, SentryTracer *tracer)
```

Comment:
I purposely used SentryTracer here to keep type safety. SentryEvent, SentrySpan and SentryTracer all have a SentryId property, but they're not interchangeable, so I did this to prevent any mistakes.